The LoadTests Go GBK Flink Batch job is flaky

The LoadTests Go GBK Flink Batch is failing over 50% of the time 
Please visit https://github.com/apache/beam/actions/workflows/beam_LoadTests_Go_GBK_Flink_Batch.yml?query=is%3Afailure+branch%3Amaster to see the logs.

```
Caused by: java.io.IOException: Cannot run program "docker": error=2, No such file or directory
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1128)
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1071)
	at org.apache.beam.runners.fnexecution.environment.DockerCommand.runShortCommand(DockerCommand.java:207)
	at org.apache.beam.runners.fnexecution.environment.DockerCommand.runShortCommand(DockerCommand.java:181)
```

--- a/.github/workflows/beam_LoadTests_Go_GBK_Flink_Batch.yml
+++ b/.github/workflows/beam_LoadTests_Go_GBK_Flink_Batch.yml
@@ -52,7 +52,7 @@ env:
   GCS_BUCKET: gs://beam-flink-cluster
   FLINK_DOWNLOAD_URL: https://archive.apache.org/dist/flink/flink-1.17.0/flink-1.17.0-bin-scala_2.12.tgz
   HADOOP_DOWNLOAD_URL: https://repo.maven.apache.org/maven2/org/apache/flink/flink-shaded-hadoop-2-uber/2.8.3-10.0/flink-shaded-hadoop-2-uber-2.8.3-10.0.jar
-  FLINK_TASKMANAGER_SLOTS: 1
+  FLINK_TASKMANAGER_SLOTS: 5
   DETACHED_MODE: true
   HARNESS_IMAGES_TO_PULL: gcr.io/apache-beam-testing/beam-sdk/beam_go_sdk:latest
   JOB_SERVER_IMAGE: gcr.io/apache-beam-testing/beam_portability/beam_flink1.17_job_server:latest
@@ -98,6 +98,8 @@ jobs:
       - name: Start Flink with parallelism 5
         env:
           FLINK_NUM_WORKERS: 5
+          HIGH_MEM_MACHINE: n1-highmem-16
+          HIGH_MEM_FLINK_PROPS: flink:taskmanager.memory.process.size=16g,flink:taskmanager.memory.flink.size=12g,flink:taskmanager.memory.jvm-overhead.max=4g,flink:jobmanager.memory.process.size=6g,flink:jobmanager.memory.jvm-overhead.max= 2g,flink:jobmanager.memory.flink.size=4g
         run: |
           cd ${{ github.workspace }}/.test-infra/dataproc; ./flink_cluster.sh create
       # The env variables are created and populated in the test-arguments-action as "<github.job>_test_arguments_<argument_file_paths_index>"
@@ -140,6 +142,8 @@ jobs:
       - name: Restart Flink with parallelism 16
         env:
           FLINK_NUM_WORKERS: 16
+          HIGH_MEM_MACHINE: n1-highmem-16
+          HIGH_MEM_FLINK_PROPS: flink:taskmanager.memory.process.size=16g,flink:taskmanager.memory.flink.size=12g,flink:taskmanager.memory.jvm-overhead.max=4g,flink:jobmanager.memory.process.size=6g,flink:jobmanager.memory.jvm-overhead.max= 2g,flink:jobmanager.memory.flink.size=4g
         run: |
           cd ${{ github.workspace }}/.test-infra/dataproc; ./flink_cluster.sh restart
       - name: run GBK Flink Batch Go Load Test 4 (fanout 4)

--- a/.github/workflows/load-tests-pipeline-options/go_GBK_Flink_Batch_100b.txt
+++ b/.github/workflows/load-tests-pipeline-options/go_GBK_Flink_Batch_100b.txt
@@ -16,7 +16,7 @@
 
 --influx_namespace=flink
 --influx_measurement=go_batch_gbk_2
---input_options=''{\"num_records\":20000000,\"key_size\":10,\"value_size\":90}''
+--input_options=''{\"num_records\":1000000,\"key_size\":10,\"value_size\":90}''
 --iterations=1
 --fanout=1
 --parallelism=5

--- a/.github/workflows/load-tests-pipeline-options/go_GBK_Flink_Batch_100kb.txt
+++ b/.github/workflows/load-tests-pipeline-options/go_GBK_Flink_Batch_100kb.txt
@@ -19,7 +19,7 @@
 --iterations=1
 --fanout=1
 --parallelism=5
---input_options=''{\"num_records\":20000,\"key_size\":10000,\"value_size\":90000}''
+--input_options=''{\"num_records\":1000,\"key_size\":10000,\"value_size\":90000}''
 --endpoint=localhost:8099
 --environment_type=DOCKER
 --environment_config=gcr.io/apache-beam-testing/beam-sdk/beam_go_sdk:latest

--- a/.github/workflows/load-tests-pipeline-options/go_GBK_Flink_Batch_10b.txt
+++ b/.github/workflows/load-tests-pipeline-options/go_GBK_Flink_Batch_10b.txt
@@ -16,7 +16,7 @@
 
 --influx_namespace=flink
 --influx_measurement=go_batch_gbk_1
---input_options=''{\"num_records\":200000000,\"key_size\":1,\"value_size\":9}''
+--input_options=''{\"num_records\":5000000,\"key_size\":1,\"value_size\":9}''
 --iterations=1
 --fanout=1
 --parallelism=5

--- a/.github/workflows/load-tests-pipeline-options/go_GBK_Flink_Batch_Fanout_4.txt
+++ b/.github/workflows/load-tests-pipeline-options/go_GBK_Flink_Batch_Fanout_4.txt
@@ -19,7 +19,7 @@
 --iterations=1
 --fanout=4
 --parallelism=16
---input_options=''{\"num_records\":5000000,\"key_size\":10,\"value_size\":90}''
+--input_options=''{\"num_records\":100000,\"key_size\":10,\"value_size\":90}''
 --endpoint=localhost:8099
 --environment_type=DOCKER
 --environment_config=gcr.io/apache-beam-testing/beam-sdk/beam_go_sdk:latest

--- a/.github/workflows/load-tests-pipeline-options/go_GBK_Flink_Batch_Fanout_8.txt
+++ b/.github/workflows/load-tests-pipeline-options/go_GBK_Flink_Batch_Fanout_8.txt
@@ -19,7 +19,7 @@
 --iterations=1
 --fanout=8
 --parallelism=16
---input_options=''{\"num_records\":2500000,\"key_size\":10,\"value_size\":90}''
+--input_options=''{\"num_records\":100000,\"key_size\":10,\"value_size\":90}''
 --endpoint=localhost:8099
 --environment_type=DOCKER
 --environment_config=gcr.io/apache-beam-testing/beam-sdk/beam_go_sdk:latest

--- a/.github/workflows/load-tests-pipeline-options/go_GBK_Flink_Batch_Reiteration_10KB.txt
+++ b/.github/workflows/load-tests-pipeline-options/go_GBK_Flink_Batch_Reiteration_10KB.txt
@@ -19,7 +19,7 @@
 --iterations=4
 --fanout=1
 --parallelism=5
---input_options=''{\"num_records\":20000000,\"key_size\":10,\"value_size\":90,\"num_hot_keys\":200,\"hot_key_fraction\":1}''
+--input_options=''{\"num_records\":1000000,\"key_size\":10,\"value_size\":90,\"num_hot_keys\":200,\"hot_key_fraction\":1}''
 --endpoint=localhost:8099
 --environment_type=DOCKER
 --environment_config=gcr.io/apache-beam-testing/beam-sdk/beam_go_sdk:latest

